pkg/test/settings.go

```diff
@@ -31,11 +30,11 @@ func Settings(overrides ...settings.Settings) *settings.Settings {
 			panic(fmt.Sprintf("Failed to merge pod options: %s", err))
 		}
 	}
-	if options.BatchMaxDuration == nil {
-		options.BatchMaxDuration = &metav1.Duration{}
+	if options.BatchMaxDuration == 0 {
+		options.BatchMaxDuration = 10 * time.Second
```

Comment:
This is going to slow testing down IIRC since these are used when doing the reconcile calls inside of provisioner

Reply:
I feel like having the max duration at 10 won't affect things. It's really the idleDuration that matters here. If we make our max too small, we may not grab all the pods that we want. Whereas the idleDuration will let the scheduling simulation begin after 1s if there's no more pods (which usually we don't add in pods in a stream in unit tests).

Reply:
Discussed offline. We no longer wait for the batcher in our testing code. At one point, we were calling the `provisioner.Reconcile` method directly but have since changed the code to just make a call directly to `provisioner.Schedule`. This means that these values shouldn't tangibly affect test time anymore.